protobufs/livekit_models.proto

```diff
@@ -193,6 +193,8 @@ message TrackInfo {
   string stream = 17;
   TimedVersion version = 18;
   repeated AudioTrackFeature audio_features = 19;
+  // server time when the track was published, in milliseconds
+  int64 published_at = 20;
```

Comment:
Can we use the `TimedVersion` ☝️

Reply:
that would certainly be easier.. let me take try that!

Reply:
that does it, thanks Raja!